ksqldb-functional-tests/src/test/resources/query-validation-tests/interval.json

```diff
@@ -0,0 +1,72 @@
+{
+  "comments": ["tests to ensure that interval is not usable in places it should not be used"],
```

Comment:
How can these intervals use in other places if the `intervalExpression` is not specified anywhere else in SqlBase.g4? I see it is only available for `identifier(...)` which are function calls.